README.md

```diff
@@ -13,6 +13,17 @@ eg: https://username-availabilty.herokuapp.com/check/facebook/yolo
 pip install -r requirements.txt
 ```
 
+### Set .env
+Rename .env.copy to .env by:
+```
+Unix or Linux:
+mv .env.copy .env
```

Comment:
not ``mv`` but ``cp`` 

If you rename, git will consider ``.env`` as deleted, which we do not want.

Reply:
sure